packages/salesforcedx-vscode-soql/src/queryDataView/queryDataViewService.ts

```diff
@@ -113,28 +117,35 @@ export class QueryDataViewService {
     return this.currentPanel.webview;
   }
 
-  protected onDidRecieveMessageHandler(message: DataViewEvent) {
+  protected onDidRecieveMessageHandler(message: DataViewEvent): void {
     const { type, format } = message;
     switch (type) {
       case 'activate':
         this.updateWebviewWith(this.queryData);
         break;
       case 'save_records':
-        this.handleSaveRecords(format!);
+        this.handleSaveRecords(format as FileFormat);
         break;
       default:
-        console.log('unknown message type from data view');
+        showAndTrackError(
+          'data_view_message_type',
+          `Dataview unable to handle message type: ${type}`
```

Comment:
@ekapner - This message here would be extremely rare...it's an error of internal communication and nothing the user could address.  It represents that we've pushed out some incompatible code that we'd likely hot-patch.

Reply:
`This is an unknown error.  Open an issue: https://github.com/forcedotcom/soql-tooling/issues/new/choose.`